packages/shared/src/components/errors/SquadLoading.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react';
+import classed from '../../lib/classed';
+
+const PlaceholderElement = classed('div', 'bg-theme-float');
+const RectangleElement = classed(PlaceholderElement, 'rounded-12');
+
+function SquadLoading(): ReactElement {
+  return (
+    <div className="flex overflow-hidden flex-col items-center pt-6 w-full max-h-page">
```

Comment:
Something I only thought about now, did you make sure this also works neatly on mobile?

Reply:
Good catch! I have now pushed a commit that displays like below:

mobile:
![image](https://user-images.githubusercontent.com/13744167/214006576-cd139b55-d828-4baa-ba9a-07862081aafa.png)

tablet:
![image](https://user-images.githubusercontent.com/13744167/214007099-1115abba-cc5b-46c3-be33-addc2e82c387.png)

laptop:
![image](https://user-images.githubusercontent.com/13744167/214007182-8cc4c7b1-9cc4-42d6-b819-4c228a8a5d3c.png)

For the Unauthorized, it works very well on mobile devices.